iocore/eventsystem/UnixEventProcessor.cc

```diff
@@ -286,11 +288,10 @@ EventProcessor::spawn_thread(Continuation *cont, const char *thr_name, size_t st
   e->mutex    = e->ethread->mutex;
   cont->mutex = e->ethread->mutex;
   {
-    ink_mutex_acquire(&dedicated_spawn_thread_mutex);
```

Comment:
It's not a real variable, by design it shouldn't be referenced again, and the name is meaningless. But, whatever.